releases/release-3.0-beta.md

```
@@ -15,93 +15,93 @@ summary: 2019年1月19日にリリースされたTiDB 3.0ベータ版は、安
+    -   クエリプランキャッシュを最適化して、ステートメントに非決定的な関数が含まれている場合にプランがキャッシュされることを保証します[#8105](https://github.com/pingcap/tidb/pull/8105)
+    -   クエリプランキャッシュを最適化して、 `DELETE` / `UPDATE` / `INSERT`のクエリプランがキャッシュされることを保証する[#8107](https://github.com/pingcap/tidb/pull/8107)
+    -   クエリプランキャッシュを最適化して、 `DEALLOCATE`文を実行するときに対応するプランを削除します。 [#8332](https://github.com/pingcap/tidb/pull/8332)
+    -   クエリプランキャッシュを最適化し、メモリ使用量を制限することで、過剰なプランをキャッシュすることによって引き起こされる TiDB OOM の問題を回避します[#8339](https://github.com/pingcap/tidb/pull/8339)
+    -   `Prepare`文を最適化して、 `ORDER BY` / `GROUP BY` / `LIMIT`句の`?`プレースホルダの使用をサポートする [#8206](https://github.com/pingcap/tidb/pull/8206)
 -   権限管理
-    -   `ANALYZE`文権限チェックを追加する [＃8486](https://github.com/pingcap/tidb/pull/8486)
-    -   `USE`文権限チェックを追加する [＃8414](https://github.com/pingcap/tidb/pull/8418)
-    -   `SET GLOBAL`文権限チェックを追加する [＃8837](https://github.com/pingcap/tidb/pull/8837)
-    -   `SHOW PROCESSLIST`文権限チェックを追加する [＃7858](https://github.com/pingcap/tidb/pull/7858)
+    -   `ANALYZE`文権限チェックを追加する [#8486](https://github.com/pingcap/tidb/pull/8486)
+    -   `USE`文権限チェックを追加する [#8414](https://github.com/pingcap/tidb/pull/8418)
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 3

(MD030, list-marker-space)